src/Commands/CloneCommand.php

```diff
@@ -36,29 +35,34 @@ public function invokeClone($origin, $target, array $options = [])
              ->setOriginEnv(explode('.', $origin)[1])
              ->setTargetEnv($target)
              ->setOperations($this->filterOperations($options));
-        array_map(
-            [$this, 'doOperation'],
-            $this->getOperations()
-        );
+        $this->doOperations($this->getOperations());
     }
 
     /**
-     * Prepare operation and execute.
-     *
-     * @param $operation
+     * @param array $operations
      *
      * @return bool
      */
-    protected function doOperation($operation = '')
+    protected function doOperations(array $operations = [])
```

Comment:
The refactor of this method makes the progress bar output smoother. I also have been thinking how we can abstract this whole concept of running a workflow with progress bar output so folks could just do `$this->io->workflowOutput($workflow)` or something like that.